web_responsive/views/inherited_view_users_form_simple_modif.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+
+<!--
+    Copyright 2018
+    @author Alexanre Díaz <dev@redneboa.es>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+    <record id="view_users_form_simple_modif" model="ir.ui.view">
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form_simple_modif" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='email']" position="after">
```

Comment:
IMO Directly use `field` here

Reply:
@Yajo Yes, but if you Get `field` directly then use `<field>` first, if not get then it's Ok.

here, First Priorty use to direct `field` and if not get then use`xpath`.